### ISSUE TITLE
[PROPOSAL] Why are Zero-Knowledge proofs important for privacy?

### ISSUE THREAD

**Issue by nickatnight23** [chainstack/developer-hub-content#20]
- Content is:
  [ ] Coding tutorial
  [x] Informational content
  [ ] Other (describe)

- Which blockchain protocol is this about?:
  Ethereum
- Description (two or three phrases describing what is it about):
Zero-Knowledge proofs are used in a way that one party can prove a claim to another party without revealing all the details to the verifying party.  Zero-Knowledge proofs are powerful because it gives the option to add a security layer on blockchains dealing with credit card payments, banks, and governments just to name a few. Information on a public blockchain can be seen by anyone and by using Zero-Knowledge proofs some of that data can be protected. I believe that there are a lot of great use cases for the technology to be implemented.
- Have you published articles in a blog before? If so, please include some links below 👇
https://dev.to/tnickatnite23/smart-contracts-376c
https://dev.to/tricadev23/ipfs-or-http-2ln4

Proposed Outline

- What are  Zero-Knowledge proofs
- Why are Zero-Knowledge proofs important for privacy
- Good reasons to implement the protocol
- Brief mention of bad actors and ramifications
- Examples of Zero-Knowledge proofs
- Conclusion
- Resources

**Comment by Genesis3800** [comment]
Hey Nick,
we are glad to have you in our developer hub program. Before we take up your proposal internally, however, we would like to see a sort of table of contents for your proposed article, to better understand what you plan to write on.

Check out this link for some inspiration-
https://github.com/chainstack/developer-hub-content/issues/18

**Comment by nickatnight23** [comment]
I just updated the proposal with an outline.

**Comment by Genesis3800** [comment]
Hey @nickatnight23 ,
we discussed your proposal internally.
I regret to inform you that we are not at the moment looking to add an article on ZK proofs to our blog at the moment. I would like to thank you again for your interest in our program. 
You are invited to apply again if you have any new ideas.

We typically look for articles that have some code in them. Here are a couple of articles we recently released-

https://chainstack.com/querying-pending-transactions-with-graphql/

https://chainstack.com/using-chainlink-data-feeds-with-foundry/
